src/components/reactions.css

```
@@ -1,44 +1,44 @@
 /* Revert reactions */
-.reaction-2A2y9y {
+.reaction-3vwAF2 {
   border-radius: .25rem;
   background: var(--background-modifier-selected);
 
   border: none;
 }
 
-.mentioned-xhSam7 .reaction-2A2y9y,
-.mentioned-xhSam7 .reaction-2A2y9y:hover {
+.mentioned-Tre-dv .reaction-3vwAF2,
+.mentioned-Tre-dv .reaction-3vwAF2:hover {
     background-color:var(--background-modifier-accent);
 }
 
-.reaction-2A2y9y .reactionCount-1zkLcN {
+.reaction-3vwAF2 .reactionCount-26U4As {
   color: var(--text-muted); /* A11Y-- */
 }
 
-.reaction-2A2y9y .reactionInner-9eVHJa {
+.reaction-3vwAF2 .reactionInner-YJjOtT {
   padding: 0 .375rem;
 }
 
 /* Self reactions */
-.reaction-2A2y9y.reactionMe-3I9gFK .reactionCount-1zkLcN {
+.reaction-3vwAF2.reactionMe-1PwQAc .reactionCount-26U4As {
   color: var(--brand-experiment); /* A11Y-- */
 }
 
-.mentioned-xhSam7 .reaction-2A2y9y.reactionMe-wv5HKu,
-.mentioned-xhSam7 .reactionMe-3I9gFK.reaction-2A2y9y:hover,
-.reaction-2A2y9y.reactionMe-3I9gFK {
+.mentioned-Tre-dv .reaction-3vwAF2.reactionMe-1PwQAc,
+.mentioned-Tre-dv .reactionMe-1PwQAc.reaction-3vwAF2:hover,
+.reaction-3vwAF2.reactionMe-1PwQAc {
```

Wouldn't it be a better idea to just remove the salts altogether? This is bound to happen again.

For example:
```suggestion
[class*="mentioned-"] [class*="reactionMe-"][class*="reaction-"],
[class*="mentioned-"] [class*="reactionMe-"][class*="reaction-"]:hover,
[class*="reactionMe-"][class*="reaction-"] {
```

Since the react classes are camelcased and always end with an hyphen, using just the name should be unique.

---

okay, as a follow up, maybe they're not unique after all... I tested this by changing all classes into the proposed format.
I used the following command: 
```
find . -type f -name "*.css" -exec sed -i 's/\.\([^-]*-\)[a-zA-Z0-9_\-]\{6\}/[class*="\1"]/g' {} +
```

the result is a bit spooky.
if you want to see for yourself, I put it here: https://github.com/Bl4Cc4t/Bl4Cc4t.github.io/blob/main/RevertRebrand.theme.css